src/main/java/com/minecolonies/blockout/controls/ItemIcon.java

```diff
@@ -15,7 +16,7 @@
  */
 public class ItemIcon extends Pane
 {
-    private static final float GUI_ITEM_Z_LEVEL = 200.0F;
+    private static final float DEFAULT_ITEMSTACK_SIZE = 16f;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Upper-case this literal "f" suffix. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS818)